### DISCUSSION TITLE
Version of the .vpr files in OpenFPGA and installation of FPGA-SPICE

### DISCUSSION THREAD

**Discussion by AyaseErii** [lnis-uofu/OpenFPGA#451] [Q&A] [answered]
Hi Xifan,

May I know the version of .vpr files in OpenFPGA? Are those vpr7? If yes, according to the documentation, it looks like I need FPGA-SPICE to run a spice simulation but there seems to be no such a tool in-built in OpenFPGA. I found some link of FPGA-SPICE but they are expired. May I know where I can get the FPGA-SPICE tool?

Thank you in advance,
Ayase

**Comment by tangxifan** [comment, marked as answer]
@AyaseErii Currently, FPGA-SPICE is still with VPR7 in an release of OpenFPGA: https://github.com/lnis-uofu/OpenFPGA/releases/tag/v0.2-alpha 
It is now in a stale branch and we are trying to bring it back to latest OpenFPGA. However, it is not a simple task. It will take some time.

**Reply by AyaseErii** [reply]
Great! Thanks for your help!
